File: old/testerum-ui-tests/src/test/java/com/testerum/JunitTesterumTests.java
package com.testerum;

import com.testerum.runner.junit5.TesterumJunitTestFactory;
import java.util.List;
import org.junit.jupiter.api.DynamicNode;
import org.junit.jupiter.api.TestFactory;

public class JunitTesterumTests {

    @TestFactory
    public List<DynamicNode> testerumTestsFactory() {

        return new TesterumJunitTestFactory("tests")
                .getTests();
    }
}

File: integration-tests/src/test/kotlin/com/testerum/JunitTesterumTests.kt
package com.testerum

import com.testerum.runner.junit5.TesterumJunitTestFactory
import org.junit.jupiter.api.DynamicNode
import org.junit.jupiter.api.TestFactory
import java.util.*

class JunitTesterumTests {

    @TestFactory
    fun testerumTestsFactory(): List<DynamicNode> {
        return TesterumJunitTestFactory("tests")
            .packagesToScan(Arrays.asList(
                "debug"
            ))
            .tests
    }
}
